datadm/backend.py

```diff
@@ -28,29 +28,27 @@ def role_end(role):
 
 
 class BackendLLMManager():
+    OPENAI_MODELS = ['gpt-4', 'gpt-4-32k', 'gpt-3.5-turbo', 'gpt-3.5-turbo-16k']
```

Comment:
Can you re-order this to be `["gpt-3.5-turbo", "gpt-4", "gpt-3.5-turbo-16k", "gpt-4-32k"]`? Then good to merge

Reply:
Realized I could change it~! I'll merge this in~